clients/itwin/src/Client.ts

```diff
@@ -210,7 +207,7 @@ export class UrlDiscoveryClient extends Client {
    * @returns URL of the discovery service.
    */
   public override async getUrl(): Promise<string> {
-    return Config.App.getString(UrlDiscoveryClient.configURL, "https://buddi.bentley.com/WebService");
+    return "https://buddi.bentley.com/WebService";

```

Comment:
Do you think this should not be configurable at all?

Reply:
Going to resolve this as there is no fix, we'll just be completely removing subsequently.